packages/css/src/screen/screen.scss

```diff
@@ -9,7 +9,9 @@
 
 .amsterdam-screen {
   @include reset;
+}
 
+.amsterdam-screen--center {
```

Comment:
Screen should always be center aligned, no? Even for full width applications

Reply:
The internal apps are aligned left if I'm not mistaken.